api/app/tests/apps/api/v1/test_private_category_endpoint.py

```diff
@@ -257,4 +257,34 @@ def test_history_view_update_handling_message(self):
         self.assertEqual(change_log_data['what'], 'UPDATED_CATEGORY')
         self.assertIsNone(change_log_data['description'])
         self.assertEqual(change_log_data['who'], self.sia_read_write_user.username)
-        self.assertIn('E-mail tekst wijziging:\n Patched handling message', change_log_data['action'])
+        self.assertIn('Servicebelofte gewijzigd naar:\n Patched handling message', change_log_data['action'])
+
+    def test_patch_category_multiple_unchanged_slo_calls(self):
+        self.client.force_authenticate(user=self.sia_read_write_user)
+
+        category = self.parent_category.children.first()
+        slo_count_at_star_of_tests = category.slo.count()
```

Comment:
Small nit: `slo_count_at_star_of_tests` -> `slo_count_at_start_of_tests`

Reply:
Fixed in next commit